app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(files_to_copy_path: str) -> None:
+    files = files_to_copy_path.split()
+    command = files[0]
+    if command == "cp":
+        if files[1] == files[2]:
+            raise ValueError("Provide another file name.")
+        try:
+            with (open(files[1], "r") as file_in,
+                  open(files[2], "w") as file_out):
+                for line in file_in:
+                    file_out.write(line)
+        except (FileNotFoundError, PermissionError):
+            raise ValueError("File not found or permission denied.")
```

Comment:
The error message 'Provide another file name.' is misleading when a `FileNotFoundError` or `PermissionError` occurs. Consider using a more descriptive message like 'File not found or permission denied.' to better inform the user of the actual issue.